tests/testthat/test-create_query_data.R

```diff
@@ -578,3 +578,21 @@ test_that("format.basket_select Test 23: formatting is correct (name specified)"
     "basket_select(name = \"My SDG\", id = NULL, scope = \"NA\", type = \"sdg\")"
   )
 })
+
+# assert_db_requirements ----
+## Test 24: assert_db_requirements_
+test_that("assert_db_requirements 24: assert_db_requirements() throws error and
+          increases code coverage when called with no `fun` arugment provided", {
+  expect_snapshot(
+    assert_db_requirements(fun = NULL),
```

Comment:
You have to specify the `queries` and `i` arguments. Otherwise, the code fails when creating the error message.